### ISSUE TITLE
Overall Loss Function

### ISSUE THREAD

**Issue by RyuJaeSeok** [VITA-Group/SinNeRF#25]
Hello! I have already read your paper and code.  But I have something curious about the overall loss function.  Because I don't know enough so sorry

you mentioned on paper : 
we reduce the weight of global structure prior λ3 and increase the weight of local texture guidance λ2.
=> I don't know it is definitely right, Since you train twice, does that mean that each lambda shrinks and grows between experiments?
.....
 In all our experiments, λ1, λ2, and λ3 are initialized to be 8, 0.1, and 0, respectively.
=>  But code is not matched to parameter setting in your code.
.....
During the training process,
we gradually decrease λ2 to 0 and increase λ3 to 0.1 with a linear function.
=> I don't know what it means, so can you tell me about this and how can I find lamda weight on your code?

**Comment by Ir1d** [comment]
Hi @RyuJaeSeok, thanks for your interest. While the loss weight scheduling seems complicated, the code included in this repo is much simpler and brings the same or slightly better results.

As shown in [README](https://github.com/VITA-Group/SinNeRF#training), The two stages include different weights for different losses.
